PSOperations/RetryOperation.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  RetryOperation.swift
+//  PSOperations
+//
+//  Created by Bart Whiteley on 9/22/15.
+//
+
+
+public class RetryOperation: GroupOperation {
+    private let operationProducer: () -> Operation
+    private let shouldRetry: (errors:[NSError], retryCount:Int) -> Bool
+    private var numberOfRetries: Int = 0
+    
+    public convenience init(@autoclosure(escaping) operation: () -> Operation, maximumNumberOfRetries: Int = 5) {
```

Comment:
It's an `@autoclosure`, which means you don't use closure syntax. `@autoclosure` lets you use this like:

    let o = RetryOperation(MyAPICallOperation(), maximumNumberOfRetries: 1)

The first parameter gets implicitly wrapped in a closure for you.